### PR TITLE
Stop using FactCheck

### PR DESCRIPTION
**Pull request by kshyatt**
Re: https://github.com/JuliaParallel/PETSc.jl/issues/49, @stevengj and @jakebolewski, how does this look to you?

### CODE REVIEW

**Comment by jakebolewski** [comment]
:100:

File: test/ksp.jl [hunk 1]
```diff
@@ -18,52 +19,48 @@ facts("\nTesting KSP") do
...
-  println("performing ksp GMRES solve")
-  x = kspg\b
-  println("finished ksp solve")
-  x_julia = A_julia\b_julia
+      @test x ≈ x_julia
 
-  for i=1:3
-    @fact x[i] --> roughly(x_julia[i])
-  end
+      debug && println("A = ", A)
```
Comment (stevengj) [inline test/ksp.jl]: I would rather just delete all of these debugging prints.

**Comment by stevengj** [comment]
Thanks for working on this!

File: test/ksp.jl (hunk 1 above)
Comment (kshyatt) [inline test/ksp.jl]: Sounds good to me, `@testset` will actually give you pretty good debugging info.

**Comment by kshyatt** [comment]
I've rebased on top of my tests for `OPTIONS` and removed the `debug` stuff like @stevengj mentioned.

**Comment by kshyatt** [comment]
This ok to merge?

**Comment by jiahao** [comment]
do it

**Pull request merged**